payments/invoice/direct.md

```diff
@@ -27,128 +27,34 @@ Pay helps improve cashflow by purchasing merchant invoices. Swedbank Pay
 receives invoice data, which is used to produce and distribute invoices to the
 consumer/end-user" %}
 
-## Introduction
+## Invoice Direct implementation flow
 
-1. Collect all purchase information and send it in a `POST` request to Swedbank
-   Pay.
-1. Include personal information (SSN and postal code) and send it to Swedbank
-   Pay.
-1. Make a new `POST` request towards Swedbank Pay to retrieve the name and
-   address of the customer.
-1. Create an authorization transaction by calculating the final price / amount.
-1. Make a third `POST` request with consumer data as input.
-1. Send a  `GET` request with the `paymentID` to get the authorization result
-1. Make a Capture by creating a `POST` request
-
-  **By making a Capture, Swedbank Pay will generate
-  the invoice to the consumer and the order is ready for shipping.**
-
-## Options before posting a payment
-
-All valid options when posting a payment with operation equal to
-`FinancingConsumer`, are described in
-[other features][financing-consumer].
-
-{:.table .table-striped}
-|               | Norway ![Norwegian flag][no-png] | Finland ![Finish flag][fi-png] | Sweden ![Swedish flag][se-png] |
-| :------------ | :------------------------------- | :----------------------------- | :----------------------------- |
-| `operation`   | `FinancingConsumer`              | `FinancingConsumer`            | `FinancingConsumer`            |
-| `currency`    | `NOK`                            | `EUR`                          | `SEK`                          |
-| `invoiceType` | `PayExFinancingNO`               | `PayExFinancingFI`             | `PayExFinancingSE`             |
+* Collect all purchase information and send it in a `POST` request to Swedbank
+  Pay. Make sure to include personal information (SSN and postal code).
+* Make a new `POST` request towards Swedbank Pay to retrieve the name and
+  address of the customer to create a purchase.
+* Create a `POST`request to retrieve the transaction status.
+* Send a  `GET` request with the `paymentID` to get the authorization result.
+* Make a Capture by creating a `POST` request.
```

Comment:
Perhaps make this into an ordered (numbered) list since the order of operations are significant?

```suggestion
1. Collect all purchase information and send it in a `POST` request to Swedbank
   Pay. Make sure to include personal information (SSN and postal code).
2. Make a new `POST` request towards Swedbank Pay to retrieve the name and
   address of the customer to create a purchase.
3. Create a `POST`request to retrieve the transaction status.
4. Send a  `GET` request with the `paymentID` to get the authorization result.
5. Make a Capture by creating a `POST` request.
```

Reply:
The reason I changed it is because Im not sure it makes sense to have it like this and only have Step 1 and Step 2. Because then we should have Step 1...Step 5. Also, the end goal is to remove the introduction completely, because I dont think we need this if we have the text structured in steps. Blir litt som "smør på flesk". So maybe the solution here would be to have all 5 steps with code examples, or write in the text that this is an ordered list of operations?

Reply:
I don't disagree with the change, I just think that the list should be ordered (numbered) instead of unordered (dotted).